config.toml.example

```diff
@@ -521,6 +521,9 @@ changelog-seen = 2
 # A descriptive string to be appended to `rustc --version` output, which is
 # also used in places like debuginfo `DW_AT_producer`. This may be useful for
 # supplementary build information, like distro-specific package versions.
+#
+# When building from a source tarball, this will default to "built from a source
+# tarball". To use pre-compiled libstd binaries, set this to "".
```

Comment:
I'm not a fan of giving this suggestion to set this to "" -- I think the implications are worth more discussion here. Can we add something like:

The Rust compiler will differentiate between different versions of itself, including based on this string, which means that if you wish to be compatible with upstream Rust you need to set this to "". However, note that if you are not *actually* compatible -- for example if you've backported patches that change behavior -- this may lead to miscompilations or other bugs.